timescaledb/how-to-guides/ingest-data/ingest-telegraf.md

```diff
@@ -5,6 +5,7 @@ keywords: [ingest, Telegraf]
 tags: [insert]
 ---
 
+import connectGrafana from "versionContent/_partials/_grafana-connect.mdx";
```

Comment:
```suggestion
import GrafanaConnect from "versionContent/_partials/_grafana-connect.mdx";
```

non-blocking suggestion

I'd like us to adopt the convention of CamelCasing the filename across the board as it will prevent any future name conflicts